ranger/data/scope.sh

```diff
@@ -94,6 +94,20 @@ handle_extension() {
             jq --color-output . "${FILE_PATH}" && exit 5
             python -m json.tool -- "${FILE_PATH}" && exit 5
             ;;
+	## 3D models
+	stl|off|dxf)
+	    [[ "${PV_IMAGE_ENABLED}" == 'True' ]] && \
+		    openscad --colorscheme="Tomorrow Night" --imgsize="1000,1000" -o "/tmp/$(basename "${FILE_PATH}").png" \
+	       <(echo "import(\"${FILE_PATH}\");")
+	    mv "/tmp/$(basename "${FILE_PATH}").png" "${IMAGE_CACHE_PATH}" && \
+	    exit 6 
+	    ;; 
+        scad|csg)
+	    [[ "${PV_IMAGE_ENABLED}" == 'True' ]] && \
+            openscad --colorscheme="Tomorrow Night" --imgsize="1000,1000" -o "/tmp/stl.png" ${FILE_PATH}; mv "/tmp/stl.png" "${IMAGE_CACHE_PATH}" && \
```

Comment:
Same issue here.

Reply:
I fixed it and squashed so there is not two seperate ops commits, or do you want me to squash it even further.

Openscad does not support exporting to Jpeg, and openscad needs the output file to be .png (only image format).

I just googled for example files, but STL is most important for me. Here are some open source stl files https://www.prusa3d.com/prusa-i3-printable-parts/ there are loads of more on thingiverse.